src/js/components/registerComp.js

```diff
@@ -0,0 +1,58 @@
+import { newContainer, newText, newMultimedia } from "../helpers/helpers";
+
+const createInputs = (formInputs) => {
+  const arrayInputs = formInputs.map((input) => {
+
+    const formFieldLabel = newText('label', input.label, ['form__field-label']);
+
+    const formFieldInput = document.createElement('input');
+    formFieldInput.classList.add('form__field-input');
+    formFieldInput.type = input.type;
+    formFieldInput.placeholder = input.placeholder;
+
+    const formField = newContainer('div', [formFieldLabel, formFieldInput], ['form__field']);
+
+    return formField;
+  });
+
+  return arrayInputs;
+
+};
+
+const registerComp = (data) => {
+
+  const formHeadline = newText('h6', data.headline, ['form__headline']);
+  const formSubheadline = newText('p', '', ['form__subheadline']);
+  //This is a "hack" to create a tag <br> within the subheadlines data
+  formSubheadline.innerHTML = data.subheadline;
+  const formInfo = newContainer('div', [formHeadline, formSubheadline], ['form__info']);
+
+  const fields = createInputs(data.fields);
+  const formGroupFields = newContainer('div', fields, ['form__group-fields']);
```

Comment:
Please use the tag `<form>`. Semantically, it is more accurate to use it.

Reply:
Done !